kubejs/server_scripts/smithing.js

```diff
@@ -0,0 +1,33 @@
+onEvent("recipes", (event) => {
+    //SMITHING RADIANT STUFF
+
+    event.smithing("createastral:bronze_ingot", "techreborn:tin_ingot", "minecraft:copper_ingot");
+    event.smithing("createastral:bronze_ingot", "minecraft:copper_ingot", "techreborn:tin_ingot");
+
+    event.smithing("createastral:radiant_helmet", "createastral:steel_helmet", "create:refined_radiance");
+    event.smithing("createastral:radiant_chestplate", "createastral:steel_chestplate", "create:refined_radiance");
+    event.smithing("createastral:radiant_leggings", "createastral:steel_leggings", "create:refined_radiance");
+    event.smithing("createastral:radiant_boots", "createastral:steel_boots", "create:refined_radiance");
+    event.smithing("createastral:radiant_sword", "minecraft:netherite_sword", "create:refined_radiance");
+    event.smithing("createastral:radiant_axe", "minecraft:netherite_axe", "create:refined_radiance");
+    event.smithing("createastral:radiant_shovel", "minecraft:netherite_shovel", "create:refined_radiance");
+    event.smithing("createastral:radiant_pickaxe", "minecraft:netherite_pickaxe", "create:refined_radiance");
+
+    event.smithing("farmersdelight:cooking_pot", "farmersdelight:skillet", "minecraft:water_bucket");
+
+    // TR units
+    event.smithing("techreborn:crude_storage_unit", "#createastral:storage_unit", "createastral:t1_upgrade");
+
+    // prefix, tier
+    [
+        ["basic", 2],
+        ["advanced", 3],
+        ["industrial", 4],
+        ["quantum", 6],
+    ].forEach((arr) => {
+        const prefix = arr[0];
+        const tier = arr[1];
+        event.smithing(`techreborn:${prefix}_storage_unit`, "#createastral:storage_unit", `createastral:t${tier}_upgrade`);

```

Comment:
Would the changes made in `kubejs/server_scripts/tags/for_recipes.js` allow for a `"techreborn:quantum_storage_unit"` to be smithed with a `"createastral:t2_upgrade"` to then craft a `"techreborn:basic_storage_unit"`?

Is that desired behavior if so? It might look good in game but maybe having a tag for each tier containing what can craft that tier so you only see previous tiers in JEI and not an option to use a quantum storage unit to craft a basic might be cleaner and less confusing for a player.

Same applies to tanks.

IDK if it really matters but wanted to ask.

I'm pretty new to KubeJS wanted to start contributing to a pack that I play and enjoy!

Reply:
fixed in latest commit